src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
For fix test errors please try to do this points:
1. `npm i @babel/eslint-parser`
2. in .eslintrc.js change
`parser: "@babel/eslint-parser"`
3. In package.json check the package version
`"@babel/eslint-parser": "^7.22.15",`

Reply:
Tried to instal @babel/eslint-parser, add this parser in .eslintrc.js file and change package version in package.json
But tests still fail, how can I resolve it?😅